test/unit/Pagination.test.js

```diff
@@ -121,6 +120,38 @@ describe('Pagination', () => {
     });
   });
 
+  describe('getPage() method', () => {
+    let component;
+
+    before(() => {
+      component = shallow(<Pagination hits={4000} page="3" />);
+    });
+
+    it('should have the method "getPage()"', () => {
+      expect(component.instance().getPage).to.be.defined;
+    });
+
+    it('should return null if no arguments are passed', () => {
+      expect(component.instance().getPage()).to.be.null;
+    });
+
+    it('should return a button with "Next Page" as the default text', () => {
+      const nextPage = component.instance().getPage(2);
```

Comment:
@ktp242 the `instance()` method is also good for testing specific results when calling methods. Thanks for that. I'm just not liking how it's not a rendered component so I have to call `nextPage.props.children[0]` to get the text instead of `nextPage.text()`.